src/tabs/tab-header-bar.scss

```diff
@@ -87,11 +87,6 @@ $label-horizontal-spacing: awsui.$space-xs;
   @include styles.text-wrapping;
 }
 
-.tabs-tab-dismiss,
-.tabs-tab-action {
```

Comment:
What happened to this class? Why is it just gone without replacement?

Reply:
😕 it seems like it also could be simplified same way with a direct test id.

But if it does not cause issues right now, I will approve this revision